test/ServiceBus.Tests/ServiceBus.Tests.csproj

```diff
@@ -6,6 +6,7 @@
 
   <ItemGroup>
     <PackageReference Include="FluentAssertions" Version="$(FluentAssertionsVersion)" />
+    <PackageReference Include="StructureMap.Microsoft.DependencyInjection" Version="1.4.0" />
```

Comment:
I think I would add an explicit test project that tests integration with several containers, using theories or base classes like we use to test different providers... Converting just one test in this project seems very random

Reply:
will do.